tests/TestWireless.py

```diff
@@ -12,3 +13,26 @@ def setUp(self):
     def test_import(self):
         # if this module loads, then the import worked...
         self.assertTrue(hasattr(Wireless, 'connect'))
+
+
+class TestCMD(unittest.TestCase):
+    """
+        Tests against cmd function.
+    """
+    def setUp(self):
+        self.com = cmd('echo "test_ok"')
+        self.empty_com = cmd('echo -n')
+
+    def test_cmdcomparission(self):
+        """
+            Check if we can test against the output of
+            the current test command
+        """
+        self.assertTrue('test_ok' in self.com)
+
+    def test_cmdlen(self):
+        """
+            Check if the output is > 0 chars.
+        """
+        self.assertFalse(len(self.empty_com) > 0)
```

Comment:
Same as the other, in the actual code on Wireless.py you do:

if len(response) > 0 and 'not found' not in response:

So, in this tests i'm doing it as close as possible to it.